index.html

```diff
@@ -23,30 +23,31 @@ <h2>Search for Pokemon</h2>
             </label>
             <label>
                 Type:
-                <input required id="filter-type">
+                <input required id="filter-type_1">
             </label>
         </section>
 
         <section>
             <h2>Pokemon</h2>
             <table>
-                <thead id="pokemon-header">
+                <thead id="pokedex-header">
                     <tr>
-                        <td>Image</td>
-                        <td class="sort" data-property="pokemon">Name</td>
+                        <td id="filter-pokemon" class="sort" data-property="pokemon">Name</td>
                         <td class="sort" data-property="type_1">Type 1</td>
                         <td class="sort" data-property="type_2">Type 2</td>
                         <td class="sort" data-property="attack">Attack</td>
-                        <td class="sort" data-property="defense">Defence</td>
-                        <td class="sort" data-property="hp">HP</td>
+                    
+
                     </tr>
                 </thead>
-                <tbody id="pokemon-body"></tbody>
+                <tbody id="pokedex-body"></tbody>
             </table>
-        </section>
+        </section
```

Comment:
```suggestion
        </section>
```
Don't forget to close your html tags.